<launch>

    <node pkg="trajectory_pkg" type="trajectory_reader_node" name="trajectory_reader_node" output="screen">
        <param name="file_path" value="/home/sherif/anscer_ws/src/trajectory_pkg/data/trajectory_data.json"/>
        <param name="marker_topic" value="/trajectory_marker"/>
    </node>
</launch>